docs/windows.md

```diff
@@ -0,0 +1,227 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay
+network to forward packets between nodes. Currently VXLAN and STT tunnels are
+supported.
+
+This page shows how to install antrea-agent on Windows Nodes and register the
+Node to existing kubernetes cluster.
+
+For the detailed design of how antrea-agent works on Windows, please refer to
```

Comment:
I suggest to remove the google doc link. Finally we should create a md doc in the repo.

Reply:
Done. Wenying help to migrate the design doc. 
https://github.com/vmware-tanzu/antrea/pull/751